src/Support/HigherOrderMessageCollection.php

```diff
@@ -30,7 +30,7 @@ public function add(string $filename, int $line, string $methodName, array $argu
     public function chain(object $target): void
     {
         foreach ($this->messages as $message) {
-            $target = $message->call($target);
```

Comment:
This could be breaking or not wanted behavior.
But for me it doesn't make sense to use `null` targets.
The better/more precise check would be a `is_object()` to prevent `TypeError` at all:
> Argument 1 passed to Pest\Support\HigherOrderMessage::call() must be an object, null given, called in src/Support/HigherOrderMessageCollection.php on line 33